js/index.js

```diff
@@ -1,304 +0,0 @@
-
-// setting up mute button
-const muteButton = document.getElementById("soundoff-button");
-muteButton.onclick = function(){
-  if (mainSong.volume === 0) {
-    mainSong.volume = 0.2;
-    skillSound.volume = 1;
-    youWonSound.volume = 1;
-    bugSound.volume = 1;
-    gameOver.volume = 1;
-    muteButton.innerHTML = "Mute";
-  } else {
-    mainSong.volume = 0;
-    skillSound.volume = 0;
-    youWonSound.volume = 0;
-    bugSound.volume = 0;
-    gameOver.volume = 0;
-    muteButton.innerHTML = "Unmute";
-  }
-}
-
-// setting intro screen and bottom text 
-const introScreen = document.getElementById("start-page-div");
-const screenshottext= document.getElementById("screenshot-text");
-screenshottext.style.display="none";
-const buttons = document.getElementById("buttons-div");
-buttons.style.display="none";
-
-// setting up of the character selection buttons 
-function setPlayer(selectedPlayer){
-  player = selectedPlayer;
-  introScreen.style.display = "none";
-  buttons.style.display="flex";
-  screenshottext.style.display="block";
-  canvas.show();
-  loop();
-  counterInterval = setInterval(counter, 1000);
-  // play main song
-  mainSong.play();
-  mainSong.loop = true;
-  mainSong.volume = 0.2;
-}
-
-const player1btn = document.getElementById("player1");
-player1btn.onclick = function(){setPlayer(player1)};
-
-const player2btn = document.getElementById("player2");
-player2btn.onclick = function(){setPlayer(player2)};
-
-const player3btn = document.getElementById("player3");
-player3btn.onclick = function(){setPlayer(player3)};
-
-// setting up new game button & resetting the game 
-const newGame = document.getElementById("new-game-button");
-newGame.onclick = function(){
-  if (player === undefined) 
-    {player=player1} 
-        introScreen.style.display = "none";
-        buttons.style.display="flex";
-        screenshottext.style.display="block";
-        canvas.show();
-        score=0;
-        count=30;
-        scoreGoal = 100;
-        fill(255,255,255);
-        // reset bugs and skills , char position
-        x = 0;
-        y = 0;
-        fallingSpeed = 3;
-        bugs = [];
-        bugs2 = [];
-        skills = [];
-        skills2 = [];
-        bugsCount = 0;
-        bugs2Count = 0;
-        skillsCount = 0;
-        skills2Count = 0;
-        bugFrequency = 0.02;
-        bug2Frequency = 0;
-        skill2Frequency = -0.005;
-        // new counter
-        clearInterval(counterInterval);
-        counterInterval = setInterval(counter, 1000);
-        // restart the loop
-        loop();
-}
-
-// setting up the counter 
-let counter = function() {
-  count = count - 1; 
-  if (count <= 0) {
-    clearInterval(counterInterval);
-    count = 0;
-  }
-};
-
-// canvas setup function 
-function setup() {
-    canvas = createCanvas(1000,601);
-    canvas.hide();
-    textSize(width/30);
-    fill(255,255,255);
-    textFont('Andale Mono');
-    noLoop();
-}
-
-// canvas preload  function 
```

Comment:
I personally wouldn't have comments like these, comments should explain why code is there. this is just re-stating something that should be known to somebody using p5.js... 

But I get that in the beginning comments like this might help you navigate the code, so it makes sense to keep them. Just going forward, you generally want to avoid comments of this kind. Also, code should be documenting itself by the use of good variable names + function names / signatures.